src/main/java/de/unistuttgart/t2/uibackend/UIBackendService.java

```diff
@@ -100,17 +86,15 @@ public List<Product> getAllProducts() {
 
             }
 
-        } catch (RestClientException e) { // 404 or something like that.
-            e.printStackTrace();
-        } catch (JsonProcessingException e) { // malformed JSON, or whatever :x
+        } catch (RestClientException | JsonProcessingException e) { // malformed JSON, or whatever :x
```

Comment:
maybe remove comment and add better exception handling

Reply:
But what kind of exception handling should we do here?
Previously, there was none.
I only made changes that don't influence the runtime behavior, and Eclipse took care of the rest...

Reply:
In other cases, at least a logger was used